target/linux/generic/pending-6.6/750-net-phy-mxl-gpy-add-LED-dimming-support.patch

```diff
@@ -0,0 +1,252 @@
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1080,6 +1151,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy115_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1103,6 +1175,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1125,6 +1198,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1148,6 +1222,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1170,6 +1245,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1193,6 +1269,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+@@ -1215,6 +1292,7 @@ static struct phy_driver gpy_drivers[] = {
+ 		.get_wol	= gpy_get_wol,
+ 		.set_loopback	= gpy_loopback,
+ 		.led_brightness_set = gpy_led_brightness_set,
++		.led_max_brightness = gpy_led_max_brightness,
+ 		.led_hw_is_supported = gpy_led_hw_is_supported,
+ 		.led_hw_control_get = gpy_led_hw_control_get,
+ 		.led_hw_control_set = gpy_led_hw_control_set,
+diff --git a/drivers/net/phy/phy_device.c b/drivers/net/phy/phy_device.c
```

Comment:
This should go in a separate patch from here on.
1/2 for `phy_device.c` and `linux/phy.h`, 2/2 for `mxl-gpy.c`.

Reply:
Fixed